packages/deployer/src/server/handlers/error.ts

```diff
@@ -9,10 +9,14 @@ export function handleError(error: unknown, defaultMessage: string): Promise<Res
   const apiError = error as ApiError;
   throw new HTTPException((apiError.status || 500) as ContentfulStatusCode, {
     message: apiError.message || defaultMessage,
+    cause: apiError.cause,
   });
 }
-export function errorHandler(err: Error, c: Context): Response {
+export function errorHandler(isDev: boolean, err: Error, c: Context): Response {
```

Comment:
this is a better strategy and not a breaking change

```suggestion
export function errorHandler(err: Error, c: Context, isDev: boolean = false): Response {
```